maxim/tests/test_connection_retry_logic.py

```
@@ -0,0 +1,713 @@
+#!/usr/bin/env python3
```

Ruff (0.11.9)

Shebang is present but file is not executable

(EXE001)